.travis.yml

```diff
@@ -9,6 +9,7 @@ php:
 matrix:
   allow_failures:
     - php: nightly
+    - php: hhvm
```

Comment:
Should we add the `hhvm` test during Travis CI build?

The `hhvm` seems that it will be different from `php` version in the future.

Reply:
That's why it is allowed to fail. Maybe it can be even removed from travis build

Reply:
Hi @alexsoft, thanks for your concern and reply.

It makes sens to let this `hhvm` to be failed during Travis CI build.

Go ahead with this, and I think this PR has been already merged.